orb.yml

```diff
@@ -9,7 +9,7 @@ jobs:
         description: |
             Scan Pull Requests and Commits for sensitive findings. You must set the NIGHTFALL_API_KEY as a CircleCI project environment variable to use this orb. View this orb's source and README for usage instructions.
         docker:
-            - image: nightfallai/nightfall_code_scanner:v1.1.3
+            - image: nightfallai/nightfall_code_scanner:v1.1.4
```

Comment:
please also bump 
https://github.com/nightfallai/nightfall_circle_orb/blob/master/src/examples/scan_findings.yml#L5
and
https://github.com/nightfallai/nightfall_circle_orb/blob/master/src/jobs/scan.yml#L4

Reply:
done! bumped to 1.1.4 as well from 1.0.0